lib/codebreaker/game.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+module Codebreaker
+  class Game
+    CODE_LENGTH = 4
+    CODE_RANGE = (1..6).freeze
+    attr_accessor :input_code, :code, :name, :difficulties, :difficulty, :hints_left, :attempts_left
+    attr_reader :minuse, :plus, :none, :hints_code
+    def initialize
+      @difficulties = Codebreaker::Loader.load('difficulties')
+      @code = generate_code
+      @hints_code = @code.chars.shuffle
+      symbols
+    end
+
+    def symbols(minuse = '-', plus = '+', none = '')
+      @minuse = minuse
+      @plus = plus
+      @none = none
+    end
```

Comment:
nit: It's nice that you made parameters not required, BUT if you need to change pluses to `*` then your instance variable `@plus` will contain '*' or any symbols. So, maybe better to rename variables like: `@exact`, `@inexact` instead of `@plus`, `@minus` ?

Reply:
good point, I'm agree